LunchTime/Restaurants/MenusOloProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using LunchTime.Models;
+using LunchTime.Restaurants.MenuBrno;
+using LunchTime.Restaurants.TODO;
+
+namespace LunchTime.Restaurants
```

Comment:
Same as Brno version. What about some generic version or simple factory?

Reply:
Yes it can be, it is same comment as Pavel mentioned, let me look and I will improve the pages reusability.